staging/src/k8s.io/apiserver/pkg/util/flowcontrol/fairqueuing/queueset/queueset_test.go

```diff
@@ -659,59 +680,83 @@ func TestContextCancel(t *testing.T) {
 		t.Fatal(err)
 	}
 	qs := qsc.Complete(fq.DispatchingConfig{ConcurrencyLimit: 1})
-	counter.Add(1) // account for the goroutine running this test
+	counter.Add(1) // account for main activity of the goroutine running this test
 	ctx1 := context.Background()
 	b2i := map[bool]int{false: 0, true: 1}
-	var qnc [2][2]int32
-	req1, _ := qs.StartRequest(ctx1, &fcrequest.WorkEstimate{Seats: 1}, 1, "", "fs1", "test", "one", func(inQueue bool) { atomic.AddInt32(&qnc[0][b2i[inQueue]], 1) })
+	var qnc [2][2]int32 // counts of calls to the following queueNoteFns
```

Comment:
please don't use shortcuts unless really needed.
The name should be self-explonatory and shouldn't require comment.

[the line below:
```
queueNoteFn1 := func(inQueue bool) { atomic.AddInt32(&qnc[0][b2i[inQueue]], 1) }
```
is completely unreadable to me.

Reply:
revised